ui/app/router.js

```diff
@@ -36,6 +36,7 @@ Router.map(function () {
         this.route('config');
         this.route('edit');
       });
+      this.route('vault-usage');
```

Comment:
It seems a little redundant to label this route `vault-usage` since we're already inside vault looking at it. I think I would have expected it to be something like `usage-reporting`. I don't have strong opinions about this, just a comment

Reply:
I've updated this one!